news/1068.feature

```diff
@@ -0,0 +1,4 @@
+- Mark restapi 7 with a zcml feature flag
```

Comment:
@sneridagh as far as I know towncrier does not allow multiple entries in one file. The "-" also need to be removed since towncrier always adds them.

Reply:
This should indeed be split into multiple files: `1068.feature.1`, `1068.feature.2`, etc.

Or keep them in one file like this, if you don't mind them ending up in the same line:

```
Mark restapi 7 with a zcml feature flag: ``plonerestapi-7``.
Add a couple of additional tests for resolveuid feature reassurance
[sneridagh]
```

(Or remove the line about additional tests entirely.)

Note that in the example I added the name of the zcml feature `plonerestapi-7` explicitly, so people don't need to look in the code to see what it is called. :-)

Reply:
@mauritsvanrees @tisto done!